diff --git a/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/runtime/ReactHostImpl.java b/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/runtime/ReactHostImpl.java
index 7f773b6..2fcff25 100644
--- a/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/runtime/ReactHostImpl.java
+++ b/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/runtime/ReactHostImpl.java
@@ -260,6 +260,7 @@ public class ReactHostImpl implements ReactHost {
   TaskInterface<Void> startSurface(final ReactSurfaceImpl surface) {
     final String method = "startSurface(surfaceId = " + surface.getSurfaceID() + ")";
     log(method, "Schedule");
+    log(method, "THIS IS PATCHED VERSION OF RN");
 
     attachSurface(surface);
     return callAfterGetOrCreateReactInstance(
