diff --git a/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/runtime/ReactHostImpl.java b/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/runtime/ReactHostImpl.java
index 7f773b6..9043cfc 100644
--- a/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/runtime/ReactHostImpl.java
+++ b/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/runtime/ReactHostImpl.java
@@ -259,6 +259,7 @@ public class ReactHostImpl implements ReactHost {
   /** package */
   TaskInterface<Void> startSurface(final ReactSurfaceImpl surface) {
     final String method = "startSurface(surfaceId = " + surface.getSurfaceID() + ")";
+    log(method, "This is patched version of ReactHost.startSurface");
+    log(method, "Some random log");
     log(method, "Schedule");
 
     attachSurface(surface);
